### ISSUE TITLE
Accordion Question needs to be a button

### ISSUE THREAD

**Issue by susumantan** [agusthas/faq-accordion-card-main#1]
Clickable should be `buttons` to provide click event for keyboard user!. (Anything that is clickable should be an interactive elements)
